src/main/java/org/folio/bulkops/controller/BulkOperationController.java

```diff
@@ -66,6 +66,9 @@
 @RequiredArgsConstructor
 @Log4j2
 public class BulkOperationController implements BulkOperationsApi {
+
+  private final byte[] utf8Bom = new byte[]{(byte) 0xEF, (byte) 0xBB, (byte) 0xBF};
```

Comment:
Can it be static constant?

Reply:
Hi @khandramai , I did it at first but sonar complains it : the values inside array can be changed,  so it can be a usual constant.